Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -206,6 +206,12 @@ script:
         if (res.StatusCode === 200) {
             return 'ok';
         }
+        // 405 - Method Not Allowed.
+        // This error can raise when using CB Defence; This URL (which is used to list sessions) is theoretically not available for CB Defence. In practice it works for some cases.
+        // 401 is the status code used for 'UNAUTHORIZED' error
+        if (CB_PRODUCT === 'Defense' || res.StatusCode === 405) {
```

Comment:
I think the condition should `AND` `if (CB_PRODUCT === 'Defense' && res.StatusCode === 405)`

Reply:
its an error. will fix it.